### ISSUE TITLE
Adding as a service on Linux ( ubuntu 18.04) fails 

### ISSUE THREAD

**Issue by gigenthomas** [deepstreamIO/deepstream.io#1032]
Trying to add as a service 
sudo ./deepstream service add -c ./conf -l /var/log

Fails with the following error . The Service was never installed.

Service already exists, please uninstall first

deepstream version: 5.0.4

**Comment by autimagic** [comment]
Got the exact same - ended up installing supervisor on that machine and running deepstream this way.
With supervisor, you'll need to use the "directory" command to chdir to the directory of deepstream before executing it, ie.;

From my supervisord.conf;

[program:deepstream]
directory=/here/is/deepstream
command=/here/is/deepstream/deepstream
autostart=true
autorestart=true

... it works :-)

**Comment by yasserf** [comment]
Yeah I would recommend running the service that way, theres too many things for me to be able to maintain so I'm slowly deprecating features that are not core.

Edit: I know this isn't the best response and I should be deleting documentation instead! But as mentioned in deepstreamIO/deepstream.io#1030 I don't have the capacity to do this so I'm just doing core maintenance every few weeks now for those who are still using this. Ideally if people are finding this super useful they will create a PR to solve the issue.
